.ci/scripts/fleet-test.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+## Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+## or more contributor license agreements. Licensed under the Elastic License;
+## you may not use this file except in compliance with the Elastic License.
+
+set -euxo pipefail
+#
+# Run the functional tests for fleets using the functional-test wrapper
+#
+# Parameters:
+#   - STACK_VERSION - that's the version of the stack to be tested. Default '8.0.0-SNAPSHOT'.
+#
+
+STACK_VERSION=${1:-'8.0.0-SNAPSHOT'}
+SUITE='fleet'
+
+.ci/scripts/functional-test.sh "${SUITE}" "" "${STACK_VERSION}"
```

Comment:
As a follow-up I'm thinking to merge the METRICBEAT_VERSION and ELASTIC_AGENT_VERSION into BEAT_VERSION, (same for the stack version var), so at some point we will be able to add the forth parameter with the beat version.